partiql-ast/src/experimental/ast.rs

```diff
@@ -147,13 +147,7 @@ pub struct OnConflict {
 
 /// `CONFLICT_ACTION <action>`
 #[derive(Clone, Debug, PartialEq)]
-pub struct ConflictAction {
-    pub kind: ConflictActionKind,
-}
-
-/// `CONFLICT_ACTION <action>`
-#[derive(Clone, Debug, PartialEq)]
-pub enum ConflictActionKind {
+pub enum ConflictAction {
     DonNothing,
```

Comment:
Not relevant to these changes but noticed a typo. 
```suggestion
    DoNothing,
```

Reply:
Fixed in a new commit.